.gitmodules

```diff
@@ -1,12 +1,9 @@
-[submodule "lib/p"]
-	path = lib/p
-	url = https://github.com/ambiata/p
 [submodule "lib/disorder"]
 	path = lib/disorder
 	url = https://github.com/ambiata/disorder.hs
 [submodule "lib/x"]
 	path = lib/x
 	url = https://github.com/ambiata/x
-[submodule "lib/twine"]
-	path = lib/twine
-	url = https://github.com/ambiata/twine
+[submodule "lib/p"]
+	path = lib/p
+	url = git@github.com:ambiata/p.git
```

Comment:
P still needed as a submodule? What about X?

Reply:
There is still work in progress to remove all the submodules, first with x-optparse and then anything in disorder. Only then can i remove p entirely.

Reply:
So probably should remove all these submodules in one go.